pkg/client/userd/service.go

```diff
@@ -135,13 +135,15 @@ func (s *service) configReload(c context.Context) error {
 func (s *service) manageSessions(c context.Context, sessionServices []trafficmgr.SessionService) error {
 	// The d.quit is called when we receive a Quit. Since it
```

Comment:
Is this "d.quit" the "s.quit" or is it something else? If the former, I think we should update the comment just for clarity